setup.py

```diff
@@ -35,19 +34,8 @@
         ),
     ],
     install_requires=[
-        "click==8.1.3",
-        "h5py",
-        "microdf_python",
-        "pandas",
-        "pathlib",
         "policyengine-core",
-        "pytest",
-        "pytest-dependency",
         "pyyaml",
-        "requests",
-        "synthimpute",
-        "tables==3.9.2",
-        "tabulate",
         "tqdm",
     ],
     extras_require={
```

Comment:
for posterity, i filed #4860 to remove redundant ones here as well. no need to put in this pr though.

Reply:
Making extras list changes in this PR.